config/testgrids/openshift/redhat-openshift-ocp-release-4.5-informing.yaml

```diff
@@ -1574,6 +1601,9 @@ test_groups:
 - days_of_results: 60
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-vsphere-upi-serial-4.5
   name: release-openshift-ocp-installer-e2e-vsphere-upi-serial-4.5
+- days_of_results: 60
+  gcs_prefix: origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-4.5
+  name: release-openshift-origin-installer-e2e-aws-4.5
```

Comment:
why are origin install jobs moving from "blocking" to "informing"?

do we need a separate dashboard for origin vs ocp, too?

Reply:
Because they are not referenced by the release controller.  Azure is.

No, CI and OCP are the same.  They are one product.  OKD is a separate product.  Bulk of testing of OKD comes from CI and OCP